plane/plane-tests/tests/common/resources/pebble.rs

```diff
@@ -182,6 +186,34 @@ impl Pebble {
     }
 }
 
+/// For some reason, Docker on Mac (Sequoia) will sometimes 500 when mounting the pebble directory
+/// inside the scratch directory. For whatever reason, first starting a dummy container that mounts
+/// the scratch directory itself (i.e. the parent of the pebble directory) seems to prevent this
+/// from happening.
+#[cfg(target_os = "macos")]
+pub async fn avoid_weird_mac_bug(name: &str, scratch_dir: &Path) -> Result<()> {
+    println!(
+        "Creating dummy container for macos {}",
+        scratch_dir.to_str().unwrap()
+    );
+    let docker = Docker::connect_with_local_defaults()?;
+    let name = format!("dummy-{}", name);
+
+    let config = Config {
+        image: Some("alpine:latest".to_string()), // not pinning because the actual container doesn't matter
+        host_config: Some(bollard::service::HostConfig {
+            binds: Some(vec![format!("{}:/mount", scratch_dir.to_str().unwrap())]),
```

Comment:
Nit: I have a slight preference for `to_string_lossy()` but it's not a big deal.